yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -2409,7 +2415,8 @@ func handleReplicaDiscoveryAndValidation(replicaEndpointsFlag string) ([]srcdb.R
 
 	// Case 1: No replicas discovered and none provided
 	if len(discoveredReplicas) == 0 && len(providedEndpoints) == 0 {
-		utils.PrintAndLogfInfo("No read replicas detected. Proceeding with single-node assessment.")
+		// TODO: Uncomment this once we enable replica discovery
+		// utils.PrintAndLogfInfo("No read replicas detected. Proceeding with single-node assessment.")
 		return nil, nil
 	}
 
```

Comment:
unrelated to this PR, but case2 should be any mismatch, right? Not just providedEnpoints=0. 
Basically, what if discovered=2, provided=1?

Reply:
Thats handled by case 3 & 4 code